pkg/cli/admin/nodeimage/monitor_test.go

```diff
@@ -0,0 +1,58 @@
+package nodeimage
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestValidateMonitor(t *testing.T) {
```

Comment:
Do you maybe plan to add a more complete test on the monitor command later?

Reply:
Yes, a test for Run will be added shortly.